CMakeLists.txt

```diff
@@ -83,7 +83,7 @@ if(NOT ${GIT_RESULT})
   set(PADDLE_VERSION ${GIT_TAG_NAME})
 else()
   execute_process(
-    COMMAND ${GIT_EXECUTABLE} log -1 --format=%H
+    COMMAND ${GIT_EXECUTABLE} log -1 --format=%h
```

Comment:
Currently, if the current commit is not git-tagged, we will use the current git commit id as the version. But pip doesn't like too long file names, thus `docker build` reports errors like:

```
---> Running in f9456a652131
paddle-a3a7e76c7fdce9a75d6e6b5a57c3f189b05bd3b7-py2-none-any.whl is not a valid wheel filename.
Storing debug log for failure in /root/.pip/pip.log
The command '/bin/sh -c pip install /usr/local/opt/paddle/share/wheels/*.whl' returned a non-zero code: 1
```

The fix is to use the shortened SHA value.